internal/slack_client/slack.go

```diff
@@ -2,11 +2,12 @@ package slack_client
 
 import (
 	"context"
+	"os"
+
 	"github.com/acs-dl/slack-module-svc/internal/config"
 	"github.com/acs-dl/slack-module-svc/internal/data"
 	"github.com/slack-go/slack"
 	"gitlab.com/distributed_lab/logan/v3"
-	"os"
 )
 
 type ClientForSlack interface {
```

Comment:
Also, just `Slack`? Because `slack_client.ClientForSlack` looks strange, just slack.Client will be much more better)